ccm_web/server/src/app.module.ts

```diff
@@ -45,7 +47,13 @@ const logger = baseLogger.child({ filePath: __filename })
     CanvasModule,
     APIModule
   ],
-  providers: [UserService]
+  providers: [
+    {
+      provide: APP_INTERCEPTOR,
+      useClass: CacheControlToHeaderInterceptor
+    },
+    UserService
```

Comment:
@ssciolla I add the interceptor at in the app module instead of `main.ts`. I also feel this is clean. I tested I don't see any difference by changing the location. I still see cache header and statics are not effected

For some reason, your comment on this is not appearing in the PR only saw via Github email notification.

Reply:
Yeah, sorry, I meant to start a review, so I deleted it and re-posted it as pending. Thanks for doing that